src/components/ConnectionPanel.vue

```diff
@@ -0,0 +1,187 @@
+                </v-tooltip>
+              </v-list-item-action>
+            </v-list-item>
+          </div>
+        </v-list>
+
+        <v-divider vertical />
+        <v-tooltip bottom>
+          <template v-slot:activator="{ on }">
+            <v-btn
+              text
+              class="logout_btn px-0"
+              size="x-large"
+              v-on="on"
+              style="border-radius: 50px"
+              @click="logout"
+            >
+              <v-icon>mdi-logout</v-icon>
+            </v-btn>
+          </template>
+          <span>{{ $tc("caption.log_out", 1) }}</span>
+        </v-tooltip>
+      </v-card>
+    </template>
+    <SettingsDialog
+      v-model="settingsDialog"
+      ref="settingsDialog"
+      @close="settingsDialog = false"
+    />
+  </div>
+</template>
+<script>
+import uuidv4 from "uuid";
+import { mapGetters } from "vuex";
+import SettingsDialog from "@/components/dialogs/SettingsDialog.vue";
+
+export default {
+  name: "ConnectionPanel",
+  components: {
+    SettingsDialog,
+  },
+  props: {
+    connectionType: {
+      type: String,
+      default: () => "jira",
+    },
+  },
+  data() {
+    return {
+      connection: this.connectionType,
+      settingsDialog: false,
+    };
+  },
+  computed: {
+    ...mapGetters({
+      credentials: "auth/credentials",
+    }),
+    profileAvatar() {
+      for (const cList of Object.values(this.credentials)) {
+        if (cList.length > 0) {
+          if (cList[0].user.avatar) {
+            return cList[0].user.avatar;
+          } else {
+            return "https://www.gravatar.com/avatar/" + cList[0].user.name;
+          }
+        }
+      }
+      return "https://www.gravatar.com/avatar/" + uuidv4();
+    },
+  },
+  methods: {
+    async openAccountLink(credentialType, credential) {
+      if (credentialType === "yatt") {
+        const yattUrl = "https://app.yatt.ai/";
+        if (this.$isElectron) {
+          await this.$electronService.openExternalLink(yattUrl);
+        } else {
+          window.open(yattUrl, "_blank");
+        }
+      } else if (credentialType === "jira") {
+        const jiraUrl = credential.orgs[0].url;
+        if (this.$isElectron) {
+          await this.$electronService.openExternalLink(jiraUrl);
+        } else {
+          window.open(jiraUrl, "_blank");
+        }
+      } else if (credentialType === "testrail") {
+        const testRailUrl = `https://${credential.url}`;
+        if (this.$isElectron) {
+          await this.$electronService.openExternalLink(testRailUrl);
+        } else {
+          window.open(testRailUrl, "_blank");
+        }
+      }
+    },
+    logout() {
+      let credentials = {};
+      if (this.credentials.jira && this.credentials.testrail) {
+        this.connection === "jira"
+          ? (credentials.testrail = this.credentials.testrail)
+          : (credentials.jira = this.credentials.jira);
+      }
+      const emptyCredentials = credentials;
```

Comment:
@dockabernathy01 - why do we define a new variable here instead of just using `credentials` in the calls below?

Reply:
ok, I will fix it.